oak_functions_launcher/src/instance/native.rs

```diff
@@ -26,6 +26,17 @@ use std::{
     path::PathBuf,
 };
 
+use std::fs;
+
+fn path_exists(s: &str) -> Result<PathBuf, String> {
```

Comment:
Why are you duplicating this function between native.rs and virtualized.rs? They're exactly the same.